src/config.rs

```diff
@@ -44,6 +46,30 @@ pub struct ExportersConfig {
     pub influxdb: InfluxDBConfig,
 }
 
+#[derive(Debug, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
+#[serde(remote = "LevelFilter")]
+pub enum LevelFilterDef {
+    Off,
+    Error,
+    Warn,
+    Info,
+    Debug,
+    Trace,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct LoggerConfig {
```

Comment:
How about if logging related configurations would be defined `GeneralConfig` instead?

Reply:
I thought about this too but even it wouldn't be a big deal, I thought that it would be cleaner to have logger settings under it's own section instead of having `logger_*` configs under general. Also if the logger config gets more complex, it would be nice to keep the general config cleaner. If you still think it's better to move them under general, I can do that.

Reply:
Okay, makes sense. Can you then just move "logging" section under "general" section in the example config file?